Prepare modern access control docs for release (#15204)
PrairieLearn/PrairieLearn: docs/getting-started/index.md

![Assessment Access tab showing modern access control defaults](screenshots/12-assessment-access.png)

> For late credit, time limits, student-specific overrides, exam mode, and PrairieTest integration, see the [assessment access control documentation](../assessment/accessControl.md).

Author: ```suggestion
To learn more about the different access control options, see the [assessment access control documentation](../assessment/accessControl.md).
```
